### ISSUE TITLE
When creating a new assignment, names in drop down are random.

### ISSUE THREAD

**Issue by kgf** [ReadyResponder/ReadyResponder#556]
It would be better to sort them, so you can find who you looking for.

**Comment by umanghome** [comment]
I'll take a look and see if I can fix this.

**Comment by umanghome** [comment]
Can you provide me with the instructions on how to replicate this? I have set up the application on my local machine, don't know where to navigate to in order to create a new assignment.

**Comment by kgf** [comment]
@umanghome Thanks for your interest in the project. 
You should go to the main menu and click `Events`. You should then click in an event. I believe our seeds.rb will create one. If not, use the blue plus (+) sign to create one in a scheduled state.
Then either create or click on a task. Then click or create a requirement. OK, you're finally there !
Now use the blue button on the left hand side to create a new assignment. Attempt to choose someone from the `Person` dropdown list. This list is currently unsorted. This is what we're requesting be changed.

Also, if you could leave a comment on this issue for any objects you had to create to reproduce this, that would be helpful !
https://github.com/ReadyResponder/ReadyResponder/issues/110

Thanks again,
Kevin

**Comment by umanghome** [comment]
I'm assuming you mean using the sidebar to create a "New Availability".

To reproduce this after executing `db:seed`, I had to create two new records under "People". One of them was given the name "Abe Lincoln", and the other was given the name "Umang Galaiya".

Another issue that I noticed is, it creates one extra entry in the dropdown, whose cause I was able to trace to the code. I will be fixing it as well.

![image](https://user-images.githubusercontent.com/5698706/31576166-50d2e076-b113-11e7-9006-affd6be61896.png)

**Comment by umanghome** [comment]
I was able to create two records in "People" after executing `db:seed`, one with the name `Abe Lincoln`, and one with the name `Umang Galaiya`. 

Then I created a new event that is scheduled in the future, created a task, and then created a requirement However, when I click on "New Assignment" and it redirects me to the form, I am unable to see any names in the dropdown. However, I am able to see the names in the dropdown for "New Availability".

Am I missing something?

Edit: I was able to get the list populated. I missed adding the departments while creating the event.
